src/components/home/About.tsx

```diff
@@ -20,7 +20,7 @@ const About: FunctionComponent<{}> = () => {
         <Grid item xs={12} lg={6}>
           <Box>
             <Typography variant="h3" color={theme.palette.text.secondary} mb={10}>
-            Mobile Toolbox comprises a team of clinical colleagues, cognition researchers, measurement scientists and technical experts with proven success in multiple large-scale validation and development projects. Co-Principal Investigators Drs. Gershon and Mangravite are at Northwestern University and Sage Bionetworks respectively. Dr. Martin Silwinski of Penn State University leads the integration of the Mobile Toolbox collaboration with Mobile Monitoring of Cognitive Change (M2C2).
+           Lorem ipsum and you know the rest.
```

Comment:
Maybe a `TODO` here as well?

Reply:
This one is weird--not sure it's actually used anywhere on the site, but the original text here was the same as the body text in the About Us section in AboutUs.tsx so I copied the new body text from there just to be sure.